### ISSUE TITLE
Updating STP Strength also updates STDP Strength

### ISSUE THREAD

**Issue by simondemeule** [Xmodal/dynamiclight#114]
This was noticed in #113, but it was found out this has been present in the code for some time already. At the moment, there is a bug that makes the STP slider map to the STDP value instead. This seems to happen in the backend — the frontend (`ParamsRack.qml`) seems to map STP and STDP to their respective sliders correctly. `Facade`, `Generator` and `SpikingNet` are possible culprits.

**Comment by simondemeule** [comment]
I've confirmed that something wrong happens in the signalling between `Facade` and `Generator`. This is shown in 7efb153:

`valueChanged ( STPStrength ) (caught from Facade):	t =  1594695622096578000 	id =  0x1197175c0 	value:  "0.5314453125"`
`updateValue ( STPStrength ) (in Generator)		t =  1594695622096626000 	id =  0x70000e16b000 	 value =  QVariant(double, 0.531445)`
`updateValueFromAlias ( STDPStrength ) (in Facade):	t =  1594695622096741000 	id =  0x1197175c0 	value:  "0.5314453125"`
`valueChangedFromAlias ( STDPStrength ) (caught from Facade):	t =  1594695622097111000 	id =  0x1197175c0 	value:  "0.5314453125"`
`valueChanged ( STDPStrength ) (caught from Generator):	t =  1594695622097162000 	id =  0x1197175c0 	value:  "0.5314453125"`

**Comment by simondemeule** [comment]
It seems like `QObject`'s `setProperty` is broken. In 7efb153 I've added more debug output that shows that the call to `setProperty` on line 162 of Generator.cpp does not behave as expected. I've used the debugger to step into the `setProperty` call, and it looks like it successfully retrieves the right meta property, but the writing process goes wrong. Maybe there is something wrong with the way the `Generator` class is setup which causes this, maybe this is a lingering bug within Qt, which would be really bad. My research online didn't reveal anything relevant. Could @aalex or @sofian give this a look?
